commons-geometry-spherical/src/test/java/org/apache/commons/geometry/spherical/twod/RegionBSPTree2STest.java

```diff
@@ -838,6 +843,48 @@ public void testCircleToPolygonBoundarySize() {
         Assert.assertEquals("Clockwise boundary size", boundary, cw.getBoundarySize(), 1.0e-7);
     }
 
+    @Test
+    public void testSmallCircleToPolygon() {
+        // arrange
+        final double radius = 5.0e-8;
+        final Point2S center = Point2S.of(0.5, 1.5);
+        final int numPts = 100;
+
+        // act
+        final RegionBSPTree2S circle = circleToPolygon(center, radius, numPts, false);
+
+        // assert
+        // https://en.wikipedia.org/wiki/Spherical_cap
+        final double area = 4.0 * PlaneAngleRadians.PI * Math.pow(Math.sin(radius / 2.0), 2.0);
+        final double boundary = PlaneAngleRadians.TWO_PI * Math.sin(radius);
+
+        SphericalTestUtils.assertPointsEq(center, circle.getCentroid(), TEST_EPS);
+        Assert.assertEquals(area, circle.getSize(), TEST_EPS);
+        Assert.assertEquals(boundary, circle.getBoundarySize(), TEST_EPS);
```

Comment:
Not sure how this is passing here. Seems like its failing for me. I'm using radius of 4.78E-8 (which is about 1 foot geographically).

Reply:
The test doesn't pass on your machine?

Reply:
The difference between the actual and expected boundary sizes is 4.937270541285221E-11 so any epsilon less than that will cause a failure. TEST_EPS is set to 1e-10.

Reply:
The boundary size calculation is seems to be different on my library (consuming this library as dependency). Running units tests of this library on my machine works fine. Must be something on my end. I don't see anything odd here.